### PR TITLE
Modify service not to return DTO

### PR DESCRIPTION
**Pull request by nakedmcse**
Modify album service not to return DTO, and controller to create DTO where needed.

### CODE REVIEW

File: src/services/UserAdminService.ts [hunk 1]
```diff
@@ -22,9 +22,10 @@ export class UserAdminService extends AbstractAdminService {
         super(ipBlackListRepo, repo, fileService, settingsDao.getSetting(GlobalEnv.BASE_URL)!);
     }
 
-    public override async getAllEntries(): Promise<AdminFileEntryDto[]> {
+    public override async getAllEntries(): Promise<FileUploadModel[]> {
         const allEntries = await this.repo.getAllEntries();
-        return this.buildFileEntryDtos(allEntries.filter(entry => !entry.hasExpired));
+        return allEntries.filter(e => !e.hasExpired);
+        // return this.buildFileEntryDtos(allEntries.filter(entry => !entry.hasExpired));
```
Comment (VictoriqueMoe) [inline src/services/UserAdminService.ts:28]: remove comment

File: src/controllers/rest/impl/security/AdminController.ts [hunk 2]
```diff
@@ -8,13 +8,17 @@ import { BodyParams } from "@tsed/platform-params";
...
+import { IpBlackListRepo } from "../../../../db/repo/IpBlackListRepo.js";
 
 @Hidden()
 @Authorize("loginAuthProvider")
 @Controller("/admin")
 export class AdminController extends AbstractAdminController implements IAdminController {
-    public constructor(@Inject() private UserAdminService: UserAdminService) {
-        super(UserAdminService);
+    public constructor(
+        @Inject() private UserAdminService: UserAdminService,
```
Comment (VictoriqueMoe) [inline src/controllers/rest/impl/security/AdminController.ts:18]: uppcase vars, remove

File: src/controllers/rest/impl/security/AdminController.ts [hunk 3]
```diff
@@ -8,13 +8,17 @@ import { BodyParams } from "@tsed/platform-params";
...
+        @Inject() private BlackListRepo: IpBlackListRepo,
```
Comment (VictoriqueMoe) [inline src/controllers/rest/impl/security/AdminController.ts:19]: same as above, but also, this does not need `private`

File: src/controllers/rest/impl/security/BucketAdminController.ts [hunk 4]
```diff
@@ -16,8 +17,9 @@ export class BucketAdminController extends AbstractAdminController implements IA
     public constructor(
         @Inject() private bucketAdminService: BucketAdminService,
         @Inject() private bucketService: BucketService,
+        @Inject() private blackListRepo: IpBlackListRepo,
```
Comment (VictoriqueMoe) [inline src/controllers/rest/impl/security/BucketAdminController.ts:20]: remove `ptivate`

**Review by VictoriqueMoe — CHANGES_REQUESTED** [review]

File: src/services/UserAdminService.ts (hunk 1 above)
Comment (nakedmcse) [reply src/services/UserAdminService.ts:28]: Done

File: src/controllers/rest/impl/security/AdminController.ts (hunk 2 above)
Comment (nakedmcse) [reply src/controllers/rest/impl/security/AdminController.ts:18]: Done, necessitated other changes

File: src/controllers/rest/impl/security/AdminController.ts (hunk 3 above)
Comment (nakedmcse) [reply src/controllers/rest/impl/security/AdminController.ts:19]: Done

File: src/controllers/rest/impl/security/BucketAdminController.ts (hunk 4 above)
Comment (nakedmcse) [reply src/controllers/rest/impl/security/BucketAdminController.ts:20]: Done

**Review by VictoriqueMoe — APPROVED** [review]

**Pull request merged**